### ISSUE TITLE
Arizona instead of Phoenix in older season

### ISSUE THREAD

**Issue by dweitz43** [bradbarkhouse/mysportsfeeds-api#43]
Hey Brad,  first I wanted to thank you for all of the effort you have put into this project, I truly appreciate it a lot and I am having a lot of fun playing with the NHL data that your api's provide.

I noticed something when looking at team standings for past seasons, it is definitely not a critical issue and I am not sure if it is absolutely necessary to change, but it seems that the responses return `Arizona Coyotes` in some cases when I believe it should be `Phoenix Coyotes`. I think that the franchise didn't change their city from `Phoenix` to `Arizona` until after the `2013-2014` season, but in this response from the `division_team_standings` api for the `2007-2008` regular season, the team appears as the `Arizona Coyotes`. Again, I don't think this is a huge deal, but I thought I might mention it just in case it is something that could have been overlooked and you would want to change.

`https://www.mysportsfeeds.com/api/feed/pull/nhl/2007-2008-regular/division_team_standings.json?teamstats=none&playerstats=none`

```
{"divisionteamstandings":{"lastUpdatedOn":"2017-07-02 10:29:08 PM","division":[{"@name":"Eastern/Atlantic","teamentry":[{"team":{"ID":"10","City":"Pittsburgh","Name":"Penguins","Abbreviation":"PIT"},"rank":"1","stats":{"GamesPlayed":{"@abbreviation":"GP","#text":"82"},"stats":null}},{"team":{"ID":"7","City":"New Jersey","Name":"Devils","Abbreviation":"NJD"},"rank":"2","stats":{"GamesPlayed":{"@abbreviation":"GP","#text":"82"},"stats":null}},{"team":{"ID":"9","City":"New York","Name":"Rangers","Abbreviation":"NYR"},"rank":"3","stats":{"GamesPlayed":{"@abbreviation":"GP","#text":"82"},"stats":null}},{"team":{"ID":"6","City":"Philadelphia","Name":"Flyers","Abbreviation":"PHI"},"rank":"4","stats":{"GamesPlayed":{"@abbreviation":"GP","#text":"82"},"stats":null}},{"team":{"ID":"8","City":"New York","Name":"Islanders","Abbreviation":"NYI"},"rank":"5","stats":{"GamesPlayed":{"@abbreviation":"GP","#text":"82"},"stats":null}}]},{"@name":"Eastern/Northeast","teamentry":[{"team":{"ID":"14","City":"Montreal","Name":"Canadiens","Abbreviation":"MTL"},"rank":"1","stats":{"GamesPlayed":{"@abbreviation":"GP","#text":"82"},"stats":null}},{"team":{"ID":"13","City":"Ottawa","Name":"Senators","Abbreviation":"OTT"},"rank":"2","stats":{"GamesPlayed":{"@abbreviation":"GP","#text":"82"},"stats":null}},{"team":{"ID":"11","City":"Boston","Name":"Bruins","Abbreviation":"BOS"},"rank":"3","stats":{"GamesPlayed":{"@abbreviation":"GP","#text":"82"},"stats":null}},{"team":{"ID":"15","City":"Buffalo","Name":"Sabres","Abbreviation":"BUF"},"rank":"4","stats":{"GamesPlayed":{"@abbreviation":"GP","#text":"82"},"stats":null}},{"team":{"ID":"12","City":"Toronto","Name":"Maple Leafs","Abbreviation":"TOR"},"rank":"5","stats":{"GamesPlayed":{"@abbreviation":"GP","#text":"82"},"stats":null}}]},{"@name":"Eastern/Southeast","teamentry":[{"team":{"ID":"5","City":"Washington","Name":"Capitals","Abbreviation":"WSH"},"rank":"1","stats":{"GamesPlayed":{"@abbreviation":"GP","#text":"82"},"stats":null}},{"team":{"ID":"3","City":"Carolina","Name":"Hurricanes","Abbreviation":"CAR"},"rank":"2","stats":{"GamesPlayed":{"@abbreviation":"GP","#text":"82"},"stats":null}},{"team":{"ID":"4","City":"Florida","Name":"Panthers","Abbreviation":"FLO"},"rank":"3","stats":{"GamesPlayed":{"@abbreviation":"GP","#text":"82"},"stats":null}},{"team":{"ID":"2","City":"Atlanta","Name":"Thrashers","Abbreviation":"ATL"},"rank":"4","stats":{"GamesPlayed":{"@abbreviation":"GP","#text":"82"},"stats":null}},{"team":{"ID":"1","City":"Tampa Bay","Name":"Lightning","Abbreviation":"TBL"},"rank":"5","stats":{"GamesPlayed":{"@abbreviation":"GP","#text":"82"},"stats":null}}]},{"@name":"Western/Pacific","teamentry":[{"team":{"ID":"26","City":"San Jose","Name":"Sharks","Abbreviation":"SJS"},"rank":"1","stats":{"GamesPlayed":{"@abbreviation":"GP","#text":"82"},"stats":null}},{"team":{"ID":"29","City":"Anaheim","Name":"Ducks","Abbreviation":"ANA"},"rank":"2","stats":{"GamesPlayed":{"@abbreviation":"GP","#text":"82"},"stats":null}},{"team":{"ID":"27","City":"Dallas","Name":"Stars","Abbreviation":"DAL"},"rank":"3","stats":{"GamesPlayed":{"@abbreviation":"GP","#text":"82"},"stats":null}},{"team":{"ID":"30","City":"Arizona","Name":"Coyotes","Abbreviation":"ARI"},"rank":"4","stats":{"GamesPlayed":{"@abbreviation":"GP","#text":"82"},"stats":null}},{"team":{"ID":"28","City":"Los Angeles","Name":"Kings","Abbreviation":"LAK"},"rank":"5","stats":{"GamesPlayed":{"@abbreviation":"GP","#text":"82"},"stats":null}}]},{"@name":"Western/Northwest","teamentry":[{"team":{"ID":"25","City":"Minnesota","Name":"Wild","Abbreviation":"MIN"},"rank":"1","stats":{"GamesPlayed":{"@abbreviation":"GP","#text":"82"},"stats":null}},{"team":{"ID":"22","City":"Colorado","Name":"Avalanche","Abbreviation":"COL"},"rank":"2","stats":{"GamesPlayed":{"@abbreviation":"GP","#text":"82"},"stats":null}},{"team":{"ID":"23","City":"Calgary","Name":"Flames","Abbreviation":"CGY"},"rank":"3","stats":{"GamesPlayed":{"@abbreviation":"GP","#text":"82"},"stats":null}},{"team":{"ID":"24","City":"Edmonton","Name":"Oilers","Abbreviation":"EDM"},"rank":"4","stats":{"GamesPlayed":{"@abbreviation":"GP","#text":"82"},"stats":null}},{"team":{"ID":"21","City":"Vancouver","Name":"Canucks","Abbreviation":"VAN"},"rank":"5","stats":{"GamesPlayed":{"@abbreviation":"GP","#text":"82"},"stats":null}}]},{"@name":"Western/Central","teamentry":[{"team":{"ID":"16","City":"Detroit","Name":"Red Wings","Abbreviation":"DET"},"rank":"1","stats":{"GamesPlayed":{"@abbreviation":"GP","#text":"82"},"stats":null}},{"team":{"ID":"18","City":"Nashville","Name":"Predators","Abbreviation":"NSH"},"rank":"2","stats":{"GamesPlayed":{"@abbreviation":"GP","#text":"82"},"stats":null}},{"team":{"ID":"20","City":"Chicago","Name":"Blackhawks","Abbreviation":"CHI"},"rank":"3","stats":{"GamesPlayed":{"@abbreviation":"GP","#text":"82"},"stats":null}},{"team":{"ID":"19","City":"Columbus","Name":"Blue Jackets","Abbreviation":"CBJ"},"rank":"4","stats":{"GamesPlayed":{"@abbreviation":"GP","#text":"82"},"stats":null}},{"team":{"ID":"17","City":"St. Louis","Name":"Blues","Abbreviation":"STL"},"rank":"5","stats":{"GamesPlayed":{"@abbreviation":"GP","#text":"82"},"stats":null}}]}]}}
```

**Comment by bradbarkhouse** [comment]
Thanks @dweitz43 !

This is something we're aware of, and we'll aim to correct it when we start looking into filling out some other historical data.  Essentially, even though the overall "team" is the same, we don't track changes like change of city, name or abbreviation for each season - but we should.  This would mirror similar changes we track for players season over season.

**Comment by bradbarkhouse** [comment]
Finally corrected as part of the v1.2 upgrade!
